coordinator/clients/smart-contract-client/src/main/kotlin/net/consensys/linea/contract/l1/Web3JLineaRollupSmartContractClient.kt

```diff
@@ -112,32 +110,30 @@ class Web3JLineaRollupSmartContractClient internal constructor(
    */
   override fun submitBlobs(
     blobs: List<BlobRecord>,
-    gasPriceCaps: GasPriceCaps?
+    gasPriceCaps: GasPriceCaps?,
+    useEstimatedGas: Boolean
```

Comment:
I don't think we should change `LineaRollupSmartContractClient` interface. I suggest you creating another implementation of it instead, that would use `eth_estimateGas`